src/EditorFeatures/Core/FindReferences/FindReferencesCommandHandler.cs

```diff
@@ -39,19 +45,18 @@ internal class FindReferencesCommandHandler : ICommandHandler<FindReferencesComm
             Contract.ThrowIfNull(streamingPresenters);
             Contract.ThrowIfNull(asyncListeners);
 
-            _waitIndicator = waitIndicator;
             _synchronousPresenters = synchronousPresenters;
             _streamingPresenters = streamingPresenters;
             _asyncListener = new AggregateAsynchronousOperationListener(
                 asyncListeners, FeatureAttribute.FindReferences);
         }
 
-        public CommandState GetCommandState(FindReferencesCommandArgs args, Func<CommandState> nextHandler)
+        public VisualStudio.Commanding.CommandState GetCommandState(FindReferencesCommandArgs args)
         {
-            return nextHandler();
+            return VisualStudio.Commanding.CommandState.Undetermined;
```

Comment:
What does Undetermined mean?

Reply:
It's a declarative way of expressing "command is enabled if anybody else says so, but I personally have no opinion whatsoever", a simpler alternative to implementing IChainedComamndHandler and delegating GetCommandState to the next handler.
About 25% of Roslyn command handlers (typically those "customizing" editor commands) are like that so I assume this is pretty important case for any lang service and I'd like to make it simple.
Better naming suggestions are welcome.

Reply:
Gotcha.  Thanks!

Reply:
CommandState.NoOpinion :D   .EnabledIfSomeoneElseWantsIt :D

Reply:
@jasonmalinowski  Unspecified is better than Undetermined indeed. What about Unknown?

Reply:
ok, went with Unspecified

---
In reply to: [159562654](https://github.com/dotnet/roslyn/pull/23769#discussion_r159562654) [](ancestors = 159562654)